### ISSUE TITLE
Very low validation accuracy with pretrained models!

### ISSUE THREAD

**Issue by ghazalehtrb** [Atze00/MoViNet-pytorch#29]
Hello, thanks for the PyTorch implementation!
I noticed that the pre-trained models have very low accuracy (0.36%) on the validation set. Have the weights changed? I'm just trying the colab tutorial.

**Comment by ghazalehtrb** [comment]
Sorry I noticed the models are pretrained on kinetics not hmdb. Still I wasn't expecting such a low accuracy.
